client/src/components/header/Header.js

```diff
@@ -0,0 +1,20 @@
+import Login from './components/LoginLi/Login';
+import Register from './components/RegisterLi/Register';
+import './Header.css'
+
+const Header = (props) => {
+    return (
+        <header>
+            <ul>
+                <Login props={props} />
+                <Register props={props} />
```

Comment:
Instead of handling logic inside login and register buttons, handle them here:
```js
{!isAuthenticated && <><button onClick...>Login</button><button onClick...>Register<button></>}
```
No need to create a saparate component for these buttons

Reply:
Also make sure you use buttons for buttons. Don't add onClick on li or div. Bad practice